vms.tf

```diff
@@ -19,7 +21,7 @@ locals {
         ["${local.rwx_filestore_endpoint}:${local.rwx_filestore_path}",
           var.jump_rwx_filestore_path,
           "nfs",
-          "_netdev,auto,x-systemd.automount,x-systemd.mount-timeout=10,timeo=14,x-systemd.idle-timeout=1min,relatime,hard,rsize=1048576,wsize=1048576,vers=3,tcp,namlen=255,retrans=2,sec=sys,local_lock=none",
+          "_netdev,auto,x-systemd.automount,x-systemd.mount-timeout=10,timeo=14,x-systemd.idle-timeout=1min,relatime,hard,rsize=1048576,wsize=1048576,nfsvers=${local.protocol_version},tcp,namlen=255,retrans=2,sec=sys,local_lock=none",
```

Comment:
Is the version info needed here? The versioning is controlled on the nfs server side. If we need to force or change the default nfs version, then yes. Otherwise we don't need to include this in the mount command on the external VMs. How will the pods/etc that are using the NFS server going to do this? If this is required we'd need to tell all of the dev teams that the nfsvers value is needed in their nfs mount commands.

Reply:
`vers` or `nfsvers` info is an optional argument. 
> If no version is specified, NFS uses the highest version supported by the kernel and the mount utility. The client negotiates a suitable version with the server, trying version 4 first, version 3 second, and version 2 last.

In the current arch by removing `vers` results in NFSv3 being used as client is unable to mount `/export` with NFSv4. If we fix the mounting problem with `/export` discussed in the above comment we should be ok to remove version info from here.

Reply:
Default value for `vers` is set to 4.1. And for NetApp it is determined based on `protocol_version` defaulting to 4.1.